forc/SanityCheck/forcing_datm_sanity.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+                           #hyun-chul.lee@noaa.gov
+import matplotlib
+#matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import matplotlib.colors as mcolors
+import netCDF4 as nc
+from netCDF4 import Dataset
+import numpy as np
+import argparse
+import glob
+import os
+import yaml
+import sys
+
+def plot_zonal_ave(data1zave,data1zmax,data1zmin,lats1,data2zave,data2zmax,data2zmin,lats2,plotpath,v,sdate,d1nm,d2nm,ne):
+    #-- 
+    mindt1=np.amin(data1zmin)
+    mindt2=np.amin(data2zmin)
+    len1=lats1.size
+    len2=lats2.size
+    mindt=min([mindt1,mindt2])
+    
+    maxdt1=np.amax(data1zmax)
+    maxdt2=np.amax(data2zmax)
+    maxdt=max([maxdt1,maxdt2])
+
+    x1lat=np.reshape(lats1,len1)
+    y1ave=np.reshape(data1zave,len1)
+    y1min=np.reshape(data1zmin,len1)
+    y1max=np.reshape(data1zmax,len1)
+    x2lat=np.reshape(lats2,len2)
+    y2ave=np.reshape(data2zave,len2)
+    y2min=np.reshape(data2zmin,len2)
+    y2max=np.reshape(data2zmax,len2)
+
+    plt.subplots_adjust(hspace=0.5,wspace=0.5)
+    pn = ne + 1
+    plt.subplot(9,3,pn)
+    plt.plot(lats1,y1ave,'b')
+    plt.plot(lats2,y2ave,'r')
+    plt.plot(lats1,y1max,'b--')
+    plt.plot(lats2,y2max,'r--')
+    plt.plot(lats1,y1min,'b:')
+    plt.plot(lats2,y2min,'r:')
+    plt.xlim(-90.0,90.0)
+    plt.ylim(mindt,maxdt)
+    plt.title(v,fontsize=7,y=0.8)
+    plt.yticks(fontsize=6)
+    plt.xticks(fontsize=6)
+
+    if (pn == 1):
+        plt.suptitle('Zonal DATM of '+d1nm+'(blue) and '+d2nm+'(red) at '+sdate+' in Ave(line), Max(dashed), Min(dotted)',fontsize=8)
+
+    if (pn < 25):
+        plt.xticks([])
+    else:
+        plt.xlabel('Latitude',fontsize=6)
+
+    if (pn == 27): 
+       #plt.show()
+        fname = [plotpath+d1nm+"_"+d2nm+"_"+sdate+".png"]
+        outname = "".join(fname)
+        plt.savefig(outname)
+        plt.close()
+
+def read_2datm_var(input1, input2, v):
```

Comment:
... that's a weird way to do it! How about you make that function just read 1 variable from 1 file name and call it twice below with input1 and input2?

Reply:
The code was cleaned up and added READ.md